qiskit/providers/models/backendconfiguration.py

```diff
@@ -325,13 +325,20 @@ def __init__(self,
         self.n_uchannels = n_uchannels
         self.u_channel_lo = u_channel_lo
         self.meas_levels = meas_levels
-        self.qubit_lo_range = qubit_lo_range
-        self.meas_lo_range = meas_lo_range
+        self.qubit_lo_range = [[range[0] * 1e9, range[1] * 1e9] for range in qubit_lo_range]
```

Comment:
it's a little confusing seeing `range` as a variable and not the built in `range` method

Reply:
agreed

Reply:
@taalexander so can you fix it. also the next line.

Reply:
yes, I will return to this soon.

Reply:
Fixed in 555f7272